lib/cylc/param_expand.py

```diff
@@ -220,7 +220,7 @@ def expand_parent_params(self, parent, param_values, origin):
         then it must be a legal value for that parameter.
 
         """
-        name, p_str_list = REC_P_ALL.match(parent).groups()[:2]
+        name, p_str_list, other = REC_P_ALL.match(parent).groups()
         if not p_str_list:
             return name
```

Comment:
`return name + other`?

Reply:
I don't think it is necessary. If there is no parameter list, `name` should capture the whole string.

Reply:
agreed